releasenotes/notes/add-effective-dimension-class-44fd7661f6ba4b33.yaml

```diff
@@ -0,0 +1,11 @@
+---
+features:
+  - |
+    Introduced two new classes, ``EffectiveDimension`` and ``LocalEffectiveDimension`` 
```

Comment:
We can make these `:class:` references for Sphinx so when people read the release notes they can just click on what will be a hyperlink so they can go to the class and its documentation easily if they are interested in more.